v2/tools/generator/internal/functions/resource_conversion_function.go

```diff
@@ -20,31 +20,30 @@ import (
 //
 // Direct conversion from the hub type:
 //
-// func (<receiver> <receiverType>) Convert<From|To>(hub conversion.Hub) error {
-// 	source, ok := hub.(*<hub.Type>)
-// 	if !ok {
-// 		return fmt.Errorf("expected <hub.Type> but received %T instead", hub)
-// 	}
-// 	return <receiver>.AssignProperties<From|To><Type>(source)
-// }
+//	func (<receiver> <receiverType>) Convert<From|To>(hub conversion.Hub) error {
+//		source, ok := hub.(*<hub.Type>)
+//		if !ok {
+//			return fmt.Errorf("expected <hub.Type> but received %T instead", hub)
+//		}
+//		return <receiver>.AssignProperties<From|To><Type>(source)
+//	}
 //
-// Indirect conversion, multiple steps via an intermediate instance
+// # Indirect conversion, multiple steps via an intermediate instance
```

Comment:
I think you can remove the added `#` here and make it not have that. I had that show up in some go playground tests but fussing with the comment a bit got it to go away.

Feels like a gofmt bug to me...

Reply:
Fixed.